app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -131,9 +131,9 @@ def textual_volumes
   def textual_endpoints
     return unless @ems.connection_configurations.hawkular
 
-    [{:label => N_('Hawkular Host Name'),
+    [{:label => _('Hawkular Host Name'),
```

Comment:
@chriskacerguis We're not really consistent wrt. single / double quotes in our application.
Though what I'm trying here is to be consistent with the nearby code -- so I use single quotes.

Nevertheless, this PR is about a different problem.